subsys/net/lib/nrf_cloud/include/nrf_cloud_codec.h

```diff
@@ -234,7 +244,7 @@ int json_send_to_cloud(cJSON * const request);
  */
 cJSON *json_create_req_obj(const char *const app_id, const char *const msg_type);
 
-/** @brief Parses received REST data for an nRF Cloud error code  */
+/** @brief Parses received REST data for an nRF Cloud error code */
```

Comment:
A general note on the verb form in all function descriptions. You should always use the imperative format: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#functions

This header file seems to have most of them in the third person format.